src/main/resources/keys/private_key_rsa_4096_pkcs8-generated.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this should not be committed to source code

Reply:
is this private key?
Private key should not be used in account app module

Reply:
we will discuss with @partha-techsophy  , after that i will let you know